sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q13.sf100/explain.txt

```diff
@@ -1,227 +1,242 @@
 == Physical Plan ==
-* HashAggregate (38)
-+- Exchange (37)
-   +- * HashAggregate (36)
-      +- * Project (35)
-         +- * BroadcastHashJoin Inner BuildRight (34)
-            :- * Project (29)
-            :  +- * BroadcastHashJoin Inner BuildRight (28)
-            :     :- * Project (22)
-            :     :  +- * BroadcastHashJoin Inner BuildRight (21)
-            :     :     :- * Project (15)
-            :     :     :  +- * BroadcastHashJoin Inner BuildRight (14)
-            :     :     :     :- * Project (9)
-            :     :     :     :  +- * BroadcastHashJoin Inner BuildRight (8)
-            :     :     :     :     :- * Filter (3)
-            :     :     :     :     :  +- * ColumnarToRow (2)
-            :     :     :     :     :     +- Scan parquet default.store_sales (1)
-            :     :     :     :     +- BroadcastExchange (7)
-            :     :     :     :        +- * Filter (6)
-            :     :     :     :           +- * ColumnarToRow (5)
-            :     :     :     :              +- Scan parquet default.customer_demographics (4)
-            :     :     :     +- BroadcastExchange (13)
-            :     :     :        +- * Filter (12)
-            :     :     :           +- * ColumnarToRow (11)
-            :     :     :              +- Scan parquet default.household_demographics (10)
-            :     :     +- BroadcastExchange (20)
-            :     :        +- * Project (19)
-            :     :           +- * Filter (18)
-            :     :              +- * ColumnarToRow (17)
-            :     :                 +- Scan parquet default.customer_address (16)
-            :     +- BroadcastExchange (27)
-            :        +- * Project (26)
-            :           +- * Filter (25)
-            :              +- * ColumnarToRow (24)
-            :                 +- Scan parquet default.date_dim (23)
-            +- BroadcastExchange (33)
-               +- * Filter (32)
-                  +- * ColumnarToRow (31)
-                     +- Scan parquet default.store (30)
+* HashAggregate (41)
++- Exchange (40)
+   +- * HashAggregate (39)
+      +- * Project (38)
+         +- * BroadcastHashJoin Inner BuildRight (37)
+            :- * Project (31)
+            :  +- * BroadcastHashJoin Inner BuildRight (30)
+            :     :- * Project (24)
+            :     :  +- * BroadcastHashJoin Inner BuildRight (23)
+            :     :     :- * Project (18)
+            :     :     :  +- * SortMergeJoin Inner (17)
```

Comment:
Previously, we don't have `Sort`, do we?

Reply:
Yes, it seem `simplify expression rules` should after `ReorderJoin`.